examples/persistent-volume-provisioning/quobyte/quobyte-admin-secret.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: quobyte-admin-secret
+type: "kubernetes.io/quobyte"
```

Comment:
Yes, at the moment the Plugin fetches the secret: 

https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/quobyte/quobyte.go#L466

https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/util.go#L142

So if the type is empty the provisioning will fail.